src/Core.jl

```diff
@@ -30,12 +30,12 @@ import .OperatorsModule:
     pow,
     pow_abs,
     div,
-    log_abs,
-    log2_abs,
-    log10_abs,
-    log1p_abs,
-    sqrt_abs,
-    acosh_abs,
+    log_nan,
```

Comment:
Could these be renamed to, e.g., `safe_log` instead of `log_nan`? What do you think?

Reply:
(It might be easiest to do a regex replace for this, to save you some time)

Reply:
Sure, I can rename them.